w_smart-seq_clustering/scanpy_clustering_workflow_parameters.yaml

```diff
@@ -0,0 +1,246 @@
+barcodes: {}
```

Comment:
remove if inputs (unless that you have tried this and all works well).

Reply:
It works fine (we've been doing it all along with the json)